include/nds/arm9/math.h

```diff
@@ -120,9 +120,9 @@ static inline int32_t mulf32(int32_t a, int32_t b)
 ///
 /// @param a
 ///     20.12 positive value.
-static inline void sqrtf32_asynch(int32_t a)
+static inline void sqrtf32_asynch(uint32_t a)
 {
-    REG_SQRT_PARAM = ((uint64_t)(uint32_t)a) << 12;
+    REG_SQRT_PARAM = ((uint64_t)) << 12;
```

Comment:
Please, at least test that the code builds before sending a PR.

Reply:
I changed it to a draft for now.